frameworks/V/vweb/vweb-mongo.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM thevlang/vlang:alpine-dev
```

Comment:
You should use a fixed version here (similarly for the rest of the Dockerfiles).